TowerForge/TowerForge/Metrics/Statistics/Implemented/TotalGamesStatistic.swift

```diff
@@ -52,4 +52,23 @@ final class TotalGamesStatistic: Statistic {
           self.init(permanentValue: value, currentValue: current, maxCurrentValue: max)
       }
 
+    func merge<T: Statistic>(with that: T) -> T? {
```

Comment:
this function has multiple repetition

Reply:
Yes, initially I tried to have a default implementation within Statistics, however, StatisticsDatabase is unable to use the merge function when done that way due to some incompatibility with the (any Statistic.Type) Metatype or something. This is the most robust workaround I found.